torch/quantization/ns/utils.py

```diff
@@ -111,6 +116,9 @@ def get_node_first_input_and_output_type(
 
             return (prev_node_output_type, NodeInputOrOutputType.FP16)
 
+        elif node.target in METHS_IO_TYPE_FP32_OR_INT8:
```

Comment:
nit: do we want to merge this with FUNS_IO_TYPE_FP32_OR_INT8 in the future? since our quantize handlers would include both, even modules.
another way to structure this is 
1. special cases like dequantize, to
2. query node.target (call_method and call_function) or type(node.target) (for call_module) in the same map, and the map can be generated from quantization_patterns.py

Reply:
yes, saving that for a separate PR, ideally the mapping would be refactored away from quantization_patterns.py first so it's easier to read and extend.